### PR TITLE
Fix sources put api issue 1378 valerii

### PR DESCRIPTION
**Pull request by ValDekh**
Fix UpdateCategoryHandler and Issue with update method in Source Controller

### CODE REVIEW

**Review by vasylashka — APPROVED** [review]

File: Streetcode/Streetcode.WebApi/appsettings.Local.json [hunk 1]
```diff
@@ -7,5 +7,9 @@
     "AllowedHeaders": [ "*" ],
     "AllowedMethods": [ "*" ],
     "PreflightMaxAge": 1
+  },
+  "ReCaptcha": {
+    "SecretKey": "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe",
+    "Url": "https://www.google.com/recaptcha/api/siteverify"
   }
```
Comment (yaakymiv) [inline Streetcode/Streetcode.WebApi/appsettings.Local.json:14]: I'm not sure but that info should probably not be commited to public rep

Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/appsettings.Local.json:14]: It's ok, don't care)

**Review by Spoilsport0 — APPROVED** [review]

File: Streetcode/Streetcode.WebApi/Controllers/BaseApiController.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Controllers/BaseApiController.cs:1]: Add new line at the end of file

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Add new line at the end of file

Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Add new line at the end of file

Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Add new line at the end of file

**Review by Lazy-Lenny — APPROVED** [review]

**Pull request merged**